### DISCUSSION TITLE
Widget not showing minutes since last read

### DISCUSSION THREAD

**Discussion by aviadp81** [NightscoutFoundation/xDrip#2018] [General]
Hi,
Since I upgraded to the latest android build (xiaomi miui 13) the widget don't show me the minutes since last reading . Minutes are being shown Only when I click on it and expanding the widget. Is there a fix for that?
Attaching minimize and expanded versions.

Android version 12skq.
MIUI version 13

![IMG_20220225_110640](https://user-images.githubusercontent.com/58710255/155851222-fad2493c-eaa7-4708-b80b-e136514dd5ee.jpg)
![IMG_20220225_110713](https://user-images.githubusercontent.com/58710255/155851223-4feb113a-68a1-4e82-bd64-904ad6c34c0e.jpg)

**Comment by Navid200** [comment]
Please edit your post as well as the title and be clear and specific which Android version you are using.

**Reply by aviadp81** [reply]
Done

**Reply by Navid200** [reply]
I'm sorry, I still don't see "Android 13" anywhere.  Do you not want to mention it for some reason?  If yes, please why?

**Reply by aviadp81** [reply]
I edited it again

**Reply by Navid200** [reply]
Sorry, I thought it was Android 13
